sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -205,7 +206,7 @@ def settings(self) -> PipelineJobSettings:
         """Settings of the pipeline job.
 
         :return: Settings of the pipeline job.
-        :rtype: ~azure.ai.ml.entities.PipelineJobSettings
+        :rtype: ~azure.ai.ml.entities._job.pipeline.pipeline_job_settings.PipelineJobSettings
```

Comment:
So `pipeline_job_settings` is no longer public? will the `~` still works for private class?

Reply:
I will change it to `~azure.ai.ml.entities.PipelineJobSettings`
`~azure.ai.ml.entities_job.pipeline. pipeline_job_the settings.PipelineJobSettings` is automatically generated and is due to the current file call method being `from azure. ai. ml.entities_job.pipeline. pipeline_job_settings import PipelineJobSettings`